client/extensions/woocommerce/app/products/product-form-details-card.js

```diff
@@ -138,10 +142,18 @@ export default class ProductFormDetailsCard extends Component {
 							<FormLabel htmlFor="description">{ __( 'Description' ) }</FormLabel>
 							{ this.renderTinyMCE() }
 						</FormFieldSet>
+						{ isNumber( product.id ) && config.isEnabled( 'woocommerce/extension-reviews' ) &&
+						<ProductReviewsWidget site={ site } product={ product } /> }
```

Comment:
Maybe set this up as a variable+conditional in the render function, just to clean this up a little?